gulpfile.js

```diff
@@ -360,6 +360,13 @@ const relayCompiler = gulp.parallel(
       })
       .pipe(gulp.dest(path.join(DIST, 'relay-compiler')));
   },
+  function copyCompilerConfigSchema() {
+    return gulp
+      .src(['relay-compiler-config-schema.json'], {
```

Comment:
@tobias-tengler This should put the config schema in the root of the `relay-compiler` npm module.

<img width="449" alt="Screenshot 2024-06-23 at 8 41 47 PM" src="https://github.com/facebook/relay/assets/162735/28067924-82ce-4606-87a7-2d632cef54b5">